packaging/docker/run.sh

```diff
@@ -62,7 +62,8 @@ done
 
 export HOME="$GF_PATHS_HOME"
 
-if [ ! -z "${GF_INSTALL_PLUGINS}" ]; then
+if [ ! -z "${GF_INSTALL_PLUGINS}" ] && [ "${GF_INSTALL_PLUGINS_FORCE}" = "true" ]; then
+  echo "GF_INSTALL_PLUGINS is deprecated, please use GF_PLUGINS_PREINSTALL(for async) or GF_PLUGINS_PREINSTALL_SYNC(for sync) instead"
```

Comment:
The deprecation message should be printed if `GF_INSTALL_PLUGINS` is set (regardless if `GF_INSTALL_PLUGINS_FORCE` is set or no).

Also, some nitpick about the format:

```suggestion
  >&2 echo "\033[0;33mWARN\033[0m: GF_INSTALL_PLUGINS is deprecated. Use GF_PLUGINS_PREINSTALL or GF_PLUGINS_PREINSTALL_SYNC instead. Checkout the documentation for more info."
```

This prints to `stderr` and adds some yellow color to the WARN wording.